README.md

```diff
@@ -1,2 +1,74 @@
-# track-air-polluton-app
-This is an api to track air polution
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Track air pollution app
+
+> This is an app for tracking air pollution in a country and showing a chart of how dense the air pollution is
+
+<br> <br>
+![image](https://user-images.githubusercontent.com/58771507/157686729-960a6aa1-e2e1-4748-84e6-bfe399c0b033.png)
+
+
+## Built With
+
+- React
+- Babel
+
+## Live Demo 
+[Live Link](https://air-pollution-tracker.herokuapp.com/)
+
+## Live Video
+[Link to Video](https://drive.google.com/file/d/1iAmJkJk9HLONWYGtILpSyGYHkLoK5gmY/view?usp=sharing)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+> You will need to have git and installed in your machine
+
+### Setup <br>
+
+> To clone Run `ghttps://github.com/Ginohmk/track-air-polluton-app.git`
+
+<br>
+
+> Run `cd track-air-polluton-app.git`
+
+### Install <br>
+
+> To install all dependencies Run `npm install`
+
+### Deployment <br>
+
+> Run`npm start`(to enable webpack live server) <br>
+> Run`npm build` (to build production folder (dist) )
+
+## Author
+
+👤 **Kanu Mike**
+
+- GitHub: [@Ginohmk](https://github.com/Ginohmk)
+- Twitter: [@michotall95](https://www.twitter.com/michotall95)
+- LinkedIn: [@kanumike](https://www.linkedin.com/in/kanu-mike-497119211/)
+- Instagram: [@savy_kanu_mike](https/instagram.com/savy_kanu_mike)
+- Facebook: [@mike.kanu](https://www.facebook.com/mike.kanu)
+
+## 🤝 Contribute
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Ginohmk/calculator-UsingReact/issues)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
```

Comment:
- [ ] In the README file, please add an "Acknowledgements" section gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Thank you, I will work on this